.github/workflows/ci.yml

```diff
@@ -61,41 +26,40 @@ jobs:
           - wasm32-wasi
 
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.56.0
-          override: true
+      - uses: actions/checkout@v3
       - run: rustup target add ${{ matrix.target }}
-      - name: cargo fetch
-        uses: actions-rs/cargo@v1
-        with:
-          command: fetch
+      - run: cargo fetch
       - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --verbose --target ${{ matrix.target }} --no-default-features
+        run: >
+          cargo build
+          --verbose
+          --target ${{ matrix.target }}
+          --no-default-features
 
-  doc-links:
-    name: Nightly lint
+  bitrot:
```

Comment:
New, to make sure we test all features _somewhere_.

Reply:
Yep, and benches / examples.